packages/blocks-to-image/src/index.spec.ts

```diff
@@ -61,6 +62,7 @@ describe('Block Images', () => {
 
   afterAll(async () => {
     await blockKitRenderer.close();
+    await connectedRenderer.close();
```

Comment:
Should the `connectedRenderer` be cleaned up in the individual test where it's initialized rather than the afterAll because it's not initialized in the beforeAll (unlike `blockKitRenderer`).

Reply:
I think the problem with this generally in tests is that if a line in the test throws, it doesn't always consistently run, which is what I was trying to ensure here. I think I should probably just initialize in the beforeAll or create a new describe block, though

Reply:
Updated over in #76 where the test first appeared